### PR TITLE
feat(schema) Drop vestigial movies.title column

### PR DESCRIPTION
**Pull request by nyergler**
**What:**

Stop writing to `movies.title` and drop the column. This **must** be deployed after #6.

**Why:**

This completes the `movies` column rename from `title` ➡️ `name`.

### CODE REVIEW

**Review by thetonywu — APPROVED** [review]
LGTM!

btw normally we use the "Assign" a PR to specific people to make it clear who needs to take the next action. Often people use "Reviewers" to list who should take a look next after the current Assignee approves.

**Comment by nyergler** [comment]
Thanks, @tonywuhl; I'll adjust my use of reviewers and assignees!

**Review by pallavi — APPROVED** [review]
LGTM

**Pull request merged**
